app/views/merchants/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container fill-register">
+  <%= render 'layouts/navbar' %>
+  <% if !@merchant.errors.empty? %>
```

Comment:
this is a great way to inform your user of any errors, another option is [flash alerts](http://api.rubyonrails.org/classes/ActionDispatch/Flash.html)